package/MDAnalysis/topology/PDBParser.py

```diff
@@ -515,3 +546,132 @@ def _parse_conect(conect):
     bond_atoms = (int(conect[11 + i * 5: 16 + i * 5]) for i in
                   range(n_bond_atoms))
     return atom_id, bond_atoms
+
+
+def fetch_pdb(
+    pdb_ids=None,
```

Comment:
What happens if `pdb_ids` is `None`? I think this should be a required argument

Reply:
If there's no useful behaviour when `pdb_ids=None`, it would be better to make it a required argument (that way the error message raised would be clearer to the user than the `HTTPError`)